server/webapp/WEB-INF/rails.new/app/helpers/pipelines_helper.rb

```diff
@@ -30,7 +30,7 @@ def run_stage_label stage
 
   def trigger_message(actual_date_time, pim)
     if pim.class != com.thoughtworks.go.presentation.pipelinehistory.PreparingToScheduleInstance
-      "(Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{h pim.getApprovedBy()}</span>&nbsp;<span class='time'></span><input type='hidden' value='#{actual_date_time}'/>)"
+      "(Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{pim.getApprovedBy()}</span>&nbsp;on&nbsp;<span class='time'>#{actual_date_time.to_long_display_date_time()}</span><input type='hidden' value='#{actual_date_time}'/>)".html_safe
```

Comment:
Removing the h method was an accident. Adding the h method back in should resolve the problem as well. 

Let me know if that isn't enough, or if you'd prefer to just switch to content_tag